app/src/client/client.ts

```diff
@@ -96,11 +96,11 @@ export class NotdClient extends ServiceClient {
     return response.collectionStatistics;
   };
 
-  public getTokenRecentSales = async (registryAddress: string, tokenId: string, limit?: number, offset?: number): Promise<Resources.TokenTransfer[]> => {
+  public getTokenRecentTransfers = async (registryAddress: string, tokenId: string, limit?: number, offset?: number): Promise<Resources.TokenTransfer[]> => {
```

Comment:
this looks wrong - how does the collection page show its sales cos you've removed the old api?